up-l3/utwin/v2/README.adoc

```diff
@@ -98,32 +98,27 @@ The Platform itself (which uTwin is part of) is acting as a distributed cache fo
 
 A Topic is a cache entry that can be refreshed individually. Refreshing cache instances happens automatically as new data propagates through the network of buses
 
-uTwin instances act as a distributed networks of caches that can communicate to retrieve values without requesting a new publication, in case a uTwin instance is lacking a Topic entry in its own cache, by retrieving the last event from the cache (uTwin instance) running on the same device as the publisher of the topic). Using the analogy of a cache, it may also be useful to implement, later, a flush() command, that would remove the last known state for specific Topics, or all Topics of a given uDevice, etc.
+uTwin instances act as a distributed networks of caches that can communicate to retrieve values without requesting a new publication, in case a uTwin instance is lacking a Topic entry in its own cache, by retrieving the last event from the cache (uTwin instance) running on the same device as the publisher of the topic. Using the analogy of a cache, it may also be useful to implement, later, a flush() command, that would remove the last known state for specific Topics, or all Topics of a given uDevice, etc.
 
 
 == Interface
 The proposal defines a new Digital Twin service, named uTwin, exposing the following API:
 
 === Access to last known state
 
-*API: `rpc GetLastEvent(Topic) returns (GetLastEventResponse)`*
+*API: `rpc GetLastMessages(uprotocol.v1.UUriBatch) returns (GetLastMessagesResponse)`*
 
-`GetLastEvent()` sends back a CE that was stored in the uTwin instance. uTwin's behavior is similar to a cache, it therefore has no "obligation" to retain the data and in worst case it will simply not provide the last published event.  Applications that make use of uTwin *MUST* therefore account for the possibility that the uTwin cannot deliver the event they are requesting. This may happen in the following cases:
```

Comment:
Wondering if parts of this deleted block may still be applicable to GetLastMessages and should be kept.

Especially the last line of the block on permissions.

Reply:
Can we keep the deleted text in lieu of removing it?

Reply:
@stevenhartley Should I just keep the deleted text as is with the new paragraph copied from the proto file? or remove the new paragraph?

Reply:
I kept original text but added the protobuf description as the first couple of lines.